### PR TITLE
Updated doc re: container linking

### PR DESCRIPTION
**Pull request by Sing-Li**
clarified the first example.  fixed typo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,13 +16,13 @@ Then start Rocket.Chat linked to this mongo instance:
 
     docker run --name rocketchat --link db -d rocket.chat
 
-This will start a Rocket.Chat instance listening on the default Meteor port of 3000 on the container.
+This will start a Rocket.Chat instance listening on the default Meteor port of 3000 on the container.  You can then link a web server to front-end Rocket.Chat.
```
Comment (pierreozoux) [inline README.md]: I think this is confusing, this is always the case. That's why in this sentence, I put 'port of 3000 on the container' which describes precisely the reality.
Your sentence is true, but maybe confusing as people already know this kind of things.

**Comment by pierreozoux** [comment]
Besides the comment, I'm fine :)

File: README.md (hunk 1 above)
Comment (Sing-Li) [inline README.md]: But I think only 'hosters' (minority) can use this first sample config.

Most users trying out Rocket.Chat just want to get it running.  And there is a missing implied step to 'get at' the already running server.

Comment (Sing-Li) [inline README.md]: Having said that, I'm okay with going ahead per your suggestion - and then mod as necessary based on feedback.

**Comment by Sing-Li** [comment]
I've reverted comment to move forward.  Please merge this and update the PR at the official image.

**Comment by pierreozoux** [comment]
Ok updated, thanks!

**Comment by Sing-Li** [comment]
Please update the hash at the PR https://github.com/docker-library/official-images/pull/888/files

**Comment by pierreozoux** [comment]
Actually this is fine as it is just about the Dockerfile, the README is another PR.

**Comment by Sing-Li** [comment]
Cool. Thx.

**Pull request merged**